samples/snippets/src/test/java/com/example/contactcenterinsights/CreateAnalyzeConversationIT.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.contactcenterinsights;
+
+import static com.google.common.truth.Truth.assertThat;
+import static junit.framework.TestCase.assertNotNull;
+
+import com.google.cloud.contactcenterinsights.v1.Conversation;
+import com.google.cloud.contactcenterinsights.v1.ConversationDataSource;
+import com.google.cloud.contactcenterinsights.v1.GcsSource;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CreateAnalyzeConversationIT {
+  private static final String PROJECT_ID = System.getenv("PROJECT_ID");
+  private static final String VOICE_TRANSCRIPT_URI = System.getenv("VOICE_TRANSCRIPT_URI");
+  private static final String CHAT_TRANSCRIPT_URI = System.getenv("CHAT_TRANSCRIPT_URI");
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  private static void requireEnvVar(String varName) {
+    assertNotNull(
+      "Environment variable " + varName + " is required to perform these tests.",
+      System.getenv(varName));
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("PROJECT_ID");
+    requireEnvVar("VOICE_TRANSCRIPT_URI");
+    requireEnvVar("CHAT_TRANSCRIPT_URI");
+  }
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  public Conversation getConversation(String transcriptUri, Conversation.Medium medium) {
+    return Conversation.newBuilder()
```

Comment:
We should include an `@After` method to clean up resources (`Conversation` objects) after the tests complete.

Here's an example:
https://github.com/googleapis/java-aiplatform/blob/master/samples/snippets/src/test/java/aiplatform/CreateTrainingPipelineImageObjectDetectionSampleTest.java#L68

Reply:
Are you okay with me leaving as is at the moment with deletion in the example?
I think ultimately we want to support using @After in the same way aiplatform is, but in order for us to do that we will need a few more sample methods (create/delete w/tests) which I think are out of scope for our first batch of samples. We can revisit this and clean it up when we add those in though, because I think this is valuable to have.

Reply:
We've pivoted on the direction, I'll update this to contain two distinct samples, one for CreateConversation and one for Delete, then we will be able to support @after.